packages/trader/src/sass/app/modules/contract.scss

```diff
@@ -68,8 +68,6 @@ $CONTRACT_INFO_BOX_PADDING: 1.6em;
                     &--collapsed {
                         display: block;
                         overflow: hidden;
-                        font-size: 1rem;
```

Comment:
This font-size was applied not only to iPhone SE (as per comment above on 66 line), but also to small  Android tablets with landscape view. After removing this, iPhone SE will look like this (1st localhost, 2d - prod)
<img width="771" alt="Screenshot 2024-06-04 at 9 43 20 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/219199a4-7623-4300-88c1-d8cc97b9af64">
<img width="771" alt="Screenshot 2024-06-04 at 9 41 34 AM" src="https://github.com/binary-com/deriv-app/assets/121025168/d3525613-d6bf-42ec-86aa-1f87590f6de3">